React/Base/RCTUtils.m

```diff
@@ -286,6 +286,31 @@ id RCTNilIfNull(id value)
   return value == (id)kCFNull ? nil : value;
 }
 
+NSURL *RCTWebSocketExecutorURL(NSString *appendString)
+{
+  NSMutableString* url;
+  NSDictionary *dict = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"RCTWebSocketExecutor"];
+  if( dict )
```

Comment:
nit: see how we format the code in other places. Space after `if`, no spaces within `()`, curly brace on the same line, etc.